code/modules/projectiles/guns/rifles.dm

```diff
@@ -1243,10 +1243,59 @@
 	damage_falloff_mult = 0
 	scatter = SCATTER_AMOUNT_TIER_8
 
-
 /obj/item/weapon/gun/rifle/l42a/training
 	current_mag = /obj/item/ammo_magazine/rifle/l42a/rubber
 
+/obj/item/weapon/gun/rifle/l42a/abr40
```

Comment:
This will be meta rushed every round due to the higher damage stat. Full stop. It must be mechanically weaker than the L42, or only be allowed to fire blanks with non interchangeable magazines.

Reply:
Huh? There is no bullet damage increase. It's an L42 with bad wield delay, space for zero attachies and slightly higher melee damage. 

I am trying to get sprites for the shortmag variant and use that as the drill rifle instead, restricting it to only half mags. Unfortunately, it lacks inhand sprites so I am waiting on them as of right now. 

The current version's sprites and mag capacity will be switched over to the civilian one, which will merely be a civvie l42a that allows more attachies, but has worse wield delay and fire rate